RELEASES.md

```diff
@@ -97,6 +97,9 @@ Compatibility Notes
 - [Rustc now catches more cases of `pub_use_of_private_extern_crate`][80763]
 - [Changes in how proc macros handle whitespace may lead to panics when used
   with older `proc-macro-hack` versions. A `cargo update` should be sufficient to fix this in all cases.][84136]
+- [Deprecated internal compiler derive macros `RustcEncodable` and `RustcDecodable`.][83160]
+- [Deprecated `alloc::LayoutErr` in favor of `alloc::LayoutError`.][81767]
```

Comment:
#81767 is the PR that adjusted the deprecation since to `1.52.0`, but it might be more informative to link to #77691 which includes the FCP.